integration_test_lite.py

```diff
@@ -4,6 +4,7 @@
 import sys
 from urllib.parse import urlencode
 import uuid
+from textwrap import indent
```

Comment:
import order needs to be fixed here. https://github.com/osu-mist/style-guides/blob/master/Python.md#imports